eng/packages/http-client-csharp/generator/Azure.Generator/test/Azure.Generator.Tests.csproj

```diff
@@ -0,0 +1,24 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
how about putting the test project into the existing `TestProjects` folder?

Reply:
These are the unit tests of the generator, not a test project, therefore we should put it in the `test` directory instead of in `TestProjects`.